app/src/main/java/com/infomaniak/mail/ui/main/thread/ThreadViewModel.kt

```diff
@@ -64,24 +62,19 @@ class ThreadViewModel(application: Application) : AndroidViewModel(application)
 
     private fun updateMessages(thread: Thread) {
```

Comment:
The name is not very appropriate, you should use a name like, `fetchIncompleteMessages` 🤔